content/pages/environment/index.mdx

```diff
@@ -341,7 +341,7 @@ Additional commands are found within the six menus: File, Edit, Sketch, Debug, T
 
 ## Preferences
 
-The Processing Development Environment (PDE) is highly configurable. The most common preferences can be modified in the Preferences window, located in the File menu on Windows and Linux and in the Processing menu on Mac Os X. The full list of preferences are stored in the "preferences.txt" file. This file can be opened and edited directly only when Processing is not running. You can find the location of this file on your computer by reading the bottom-left corner of the Preferences window.
+The Processing Development Environment (PDE) is highly configurable. The most common preferences can be modified in the Preferences window, located in the File menu on Windows and Linux and the Processing menu on Mac Os X. The full list of preferences is stored in the "preferences.txt" file. This file can be opened and edited directly only when Processing is not running. You can find the location of this file on your computer by reading the bottom-left corner of the Preferences window.
```

Comment:
'macOS' is not called X anymore. Just 'macOS'

Reply:
@Arya-Gupta there is still this one thing pending. After that I'll happily merge your changes :)